lib/build_targets/package.dart

```diff
@@ -124,10 +124,6 @@ class DotnetTpk extends TizenPackage {
       copyDirectory(pluginsLibDir, libDir);
     }
 
-    final TizenManifest tizenManifest =
-        TizenManifest.parseFromXml(tizenProject.manifestFile);
```

Comment:
Is there any way to check the TargetFramework in this dart script?
In #418, I used the way to check the api version in the manifest to use "*_nui.so".

Reply:
I don't think there is. I think relying on the `api-version` value in the manifest should be enough. Also don't forget to update code in `plugins.dart` (line 92) as well.